src/Config/ConfigLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Robo\Config;
+
+use Grasmash\YamlExpander\Expander;
+
+/**
+ * Load configuration files, and fill in any property values that
+ * need to be expanded.
+ */
+class ConfigLoader implements ConfigLoaderInterface
+{
+    protected $config = [];
+    protected $source = '';
+
+    public function getSourceName()
+    {
+        return $this->source;
+    }
+
+    public function setSourceName($source)
+    {
+        $this->source = $source;
+        return $this;
+    }
+
+    public function import($data)
+    {
+        $this->config = [];
```

Comment:
Perhaps I should be implementing these classes differently, but I'm currently doing this:

```
$config = new DefaultConfig();
$yaml_config_loader = new \Robo\Config\YamlConfigLoader();
$yaml_config_loader->load($config->get('blt.root') . '/phing/build.yml');
$yaml_config_loader->load($config->get('repo.root') . '/blt/project.yml');
$yaml_config_loader->load($config->get('repo.root') . '/blt/project.local.yml');
$config = new BltConfig($yaml_config_loader->export());
```

This currently has the following problems:
* YamlConfigLoader() doesn't have access to the initial `$config` var, which is required for property expansion.
* Calling YamlConfigLoader()->load() wipes out previously loaded config.